doc/spire_agent.md

```diff
@@ -70,10 +70,12 @@ This may be useful for templating configuration files, for example across differ
 | `workload_x509_svid_key_type`     | The workload X509 SVID key type &lt;rsa-2048&vert;ec-p256&gt;                                                                                                                                                                                     | ec-p256                          |
 | `availability_target`             | The minimum amount of time desired to gracefully handle SPIRE Server or Agent downtime. This configurable influences how aggressively X509 SVIDs should be rotated. If set, must be at least 24h. See [Availability Target](#availability-target) |                                  |
 
-| experimental      | Description                                                     | Default                 |
-|:------------------|-----------------------------------------------------------------|-------------------------|
-| `named_pipe_name` | Pipe name to bind the SPIRE Agent API named pipe (Windows only) | \spire-agent\public\api |
-| `sync_interval`   | Sync interval with SPIRE server with exponential backoff        | 5 sec                   |
+| experimental               | Description                                                         | Default                 |
+|:---------------------------|---------------------------------------------------------------------|-------------------------|
+| `named_pipe_name`          | Pipe name to bind the SPIRE Agent API named pipe (Windows only)     | \spire-agent\public\api |
+| `sync_interval`            | Sync interval with SPIRE server with exponential backoff            | 5 sec                   |
+| `x509_svid_cache_max_size` | Soft limit of max number of SVIDs that would be stored in LRU cache | 1000                    |
+| `disable_lru_cache`        | Disables the SPIRE Agent LRU cache used for storing SVIDs           | false                   |
```

Comment:
doc looks good, but I'm worried if people will understand what is the difference disabling it, can you explan a little more about how it will affect? maybe telling that it go back to old approach?

Reply:
Updated the description to convey that it will use non lru cache. cc @azdagron